src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,35 @@
+package duke.command;
+
+import duke.exception.DukeException;
+import duke.storage.Storage;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+/**
+ * Is responsible for handling commands
```

Comment:
Perhaps more information could be given to explain that Command is a super class for the various types of Command objects, and that the role of a Command object is to create the correct type of Task and pass it to Ui for the correct string output to be printed.